posts/continuous-code-quality.md

```diff
@@ -0,0 +1,75 @@
+---
+authors:
+- Amine Ouali Alami
+tags:
+- continuous code quality
+- jenkinsfile
+- quality gate
+- devops
+- jenkins
+- sonarqube
+date: 2010-01-23T10:00:00.000Z
+title: Implement a Continuous Code Quality approach
+---
+
+Code quality is an essential element in a development project.
+In a traditional approach of quality, the development team should audit the code before each publication, this is not always the right approach in the long term:
+- The code review arrives late in the process, all parties are waiting for the new product.
+- The team is not necessarily aware of code quality at the time of development.
+
+
+In a continuous code quality approach, the team focuses on the new code:
+- It is difficult for the team to review the legacy of the whole project but generally delighted to fix the days issue.
+- The team is responsible for the quality of the code.
+- Go or No-Go criteria are clear and shared by everyone because they apply to the new code regardless of the context of the project.
```

Comment:
```suggestion
- Go or no-go criteria are clear and shared by everyone because they apply to the new code regardless of the context of the project.
```